package.json

```diff
@@ -9,7 +9,7 @@
   },
   "devDependencies": {
     "@autorest/autorest": "https://github.com/Azure/autorest/releases/download/autorest-3.0.6153/autorest-3.0.6153.tgz",
-    "@microsoft.azure/autorest.testserver": "^2.10.1"
+    "@microsoft.azure/autorest.testserver": "^2.10.6"
```

Comment:
Update to 2.10.7 (https://github.com/Azure/autorest.testserver/pull/123#event-2951715090)

Reply:
Done.